### ISSUE TITLE
foreign key update and delete actions are not reflected in pg_constraint table

### ISSUE THREAD

**Issue by AlexMesser** [cockroachdb/cockroach#34862]
FK's update and delete actions works fine, but not reflected in `pg_constraint` table.

Steps to reproduce:

```sql
CREATE TABLE "foo" ("id" INT PRIMARY KEY)

CREATE TABLE "bar" ("id" INT PRIMARY KEY, "fooId" INT)

CREATE INDEX "idx_fooId" ON "bar" ("fooId") 

ALTER TABLE "bar" 
     ADD CONSTRAINT "fk_foo_bar" 
     FOREIGN KEY ("fooId") REFERENCES "foo"("id") 
     ON UPDATE SET NULL
     ON DELETE SET NULL

SELECT "conname", "confupdtype", "confdeltype" FROM "pg_constraint"
```
results:

conname	| confupdtype	| confdeltype
-----------|----------------|-------------
fk_foo_bar |	a  |	a

from the postgres [docs](https://www.postgresql.org/docs/current/catalog-pg-constraint.html):
`confupdtype` - Foreign key update action code: a = no action, r = restrict, c = cascade, n = set null, d = set default
`confdeltype` - Foreign key deletion action code: a = no action, r = restrict, c = cascade, n = set null, d = set default

**Comment by BramGruneir** [comment]
Should be an easy fix, I'll take it on.

**Comment by knz** [comment]
@BramGruneir thanks :)

**Comment by knz** [comment]
Fixing in  #35052

**Comment by knz** [comment]
@AlexMesser we're fixing this in the `master` branch. @awoods187 does the fix need to be ported back to 2.1 too?
